docs/changelog/overview.mdx

```diff
@@ -12,6 +12,15 @@ This page lists changes to Flare's API.
 Release notes for the Flare Platform can be found on the [product documentation website](https://docs.flare.io/releases).
 </Note>
 
+<Update label="2025-01" description="API - January 2025">
+  Released new [/firework/v2/activities/ <Icon icon="code" size={16} />](/api-reference/v2/endpoints/activities/get-activities-).
+  You can now fetch an activity by passing the activity uid as a query parameter. This is an alternative to [/firework/v2/activities/\{index}/\{source\}/\{id\} <Icon icon="code" size={16} />](/api-reference/v2/endpoints/activities/get-activities).
+
+  Updated basic rate limiting from 2 requests per second to 4 requests per second [/concepts/rate-limits-and-quotas <Icon icon="book" size={16} />](/concepts/rate-limits-and-quotas)
```

Comment:
I don't think the slashes in the link here is pretty.

How about putting the link on "basic rate limiting" ?